float32[3] x #posicao x desejada
float32[3] y #posicao y desejada
float32[3] th #angulo desejado
float32[3] u #velocidade linear
float32[3] w #velocidade angular
float32[3] vx #velocidade desejada em x(em relacao ao campo)
float32[3] vy #velocidade desejada em y(em relacao ao campo)
int8[3] control_options #estilo de controle usado
